lib/tasks/annotate_models.rake

```diff
@@ -21,7 +21,7 @@ task annotate_models: :environment do
   options[:show_indexes] = Annotate.true?(ENV['show_indexes'])
   options[:simple_indexes] = Annotate.true?(ENV['simple_indexes'])
   options[:model_dir] = ENV['model_dir'] ? ENV['model_dir'].split(',') : ['app/models']
-  options[:root_dir] = ENV['root_dir'] ? ENV['root_dir'].split(',') : ['']
+  options[:root_dir] = ENV['root_dir'] && !ENV['root_dir'].empty? ? ENV['root_dir'].split(',') : ['']
```

Comment:
I wonder if we should fix it where options[:root_dir] being used, make it work whether it's [], [''], or ['', '',...].